### ISSUE TITLE
Long strings from drop-down fields in Preferences overlap with the drop-down arrow

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [mozilla/hubs#3824]
**[Affected Versions]:**
- Google Chrome - Version 88.0.4324.104
- Firefox Release - Version 85.0

**[Affected Platforms]:**
- Windows 10 x64.

**[Prerequisites]:**
- A room is opened on the "https://hubs.mozilla.com/" website.

**[Steps to reproduce]:**
1. Open the Preference menu from the More menu.
2. Open for example the "Misc" section.
3. Click the "Preferred camera" drop down-menu.
4. Select an option that has a long string.
5. Observe the behavior.

**[Expected results]:**
- The string is correctly displayed in the field.

**[Actual results]:**
- The string overlaps with drop-down arrow.

**[Notes]:**
- The strings length depends from a browser to another.
- Attached is a screenshot of the issue: 
![PreferencesStrings](https://user-images.githubusercontent.com/40061643/106481358-06bf7180-64b5-11eb-8fba-d5006cfca64a.PNG)

**Comment by csnktms** [comment]
https://github.com/mozilla/hubs/pull/3861

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and is no longer reproducible on "https://hubs.mozilla.com/" using App version: 1.0.0.20210220005842. Now, the drop-down input fields have a fixed width, the long strings are cut and are no longer overlapping with the buttons.
Tested on:
- Windows 10 using Firefox Release (85.0.2) and Chrome (88.0.4324.182).
- Iphone SE using Safari 14 and Chrome (87.0.4280.77).

Considering this I will close this issue as verified fixed.
